Missing dev dependency in requirements.txt

Hello! Thank you for the package!

I tried to run tests, but they failed due to missing `httpx` dependency.

Repro steps:
```
$ python3 -m venv venv
$ source venv/bin/activate
$ pip install -e .
$ pip install -r requirements.txt
$ pytest tests
ImportError while loading conftest '/home/maxim/github/fastapi-kafka/tests/conftest.py'.
tests/conftest.py:4: in <module>
    from fastapi.testclient import TestClient
venv/lib/python3.12/site-packages/fastapi/testclient.py:1: in <module>
    from starlette.testclient import TestClient as TestClient  # noqa
venv/lib/python3.12/site-packages/starlette/testclient.py:35: in <module>
    raise RuntimeError(
E   RuntimeError: The starlette.testclient module requires the httpx package to be installed.
E   You can install this with:
E       $ pip install httpx
```

System info:
```
$ python3 --version
Python 3.12.3
$ pip list
Package           Version Editable project location
----------------- ------- --------------------------------
aiokafka          0.12.0
annotated-types   0.7.0
anyio             4.7.0
async-timeout     5.0.1
bump2version      1.0.1
fastapi           0.115.6
fastapi_kafka     0.1.4   /home/maxim/github/fastapi-kafka
idna              3.10
iniconfig         2.0.0
packaging         24.2
pip               24.0
pluggy            1.5.0
pydantic          2.10.4
pydantic_core     2.27.2
pytest            8.3.4
pytest-asyncio    0.25.0
pytest-mock       3.14.0
ruff              0.2.0
sniffio           1.3.1
starlette         0.41.3
typing_extensions 4.12.2
$ uname -r
5.15.153.1-microsoft-standard-WSL2
```

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,4 +1,5 @@
 -e .
+httpx
 ruff==0.2.0
 pytest
 pytest-asyncio

